tests/common.go

```diff
@@ -629,6 +632,7 @@ func ValidateContextForPureVolumesSDK(ctx *scheduler.Context, errChan ...*chan e
 	}()
 	ginkgo.Describe(fmt.Sprintf("For validation of %s app", ctx.App.Key), func() {
 		var timeout time.Duration
+		var mountoption = []string{"nosuid"}
```

Comment:
Can we call this `requiredMountOptions` or something, to clarify that it's what we're checking for and we're not setting/overriding the PVC mount options?

Reply:
Done